app/views/devise/sessions/new.html.erb

```diff
@@ -0,0 +1,29 @@
+
+<div class="card p-0 col-md-4 col-xl-3">
+  <%= simple_form_for(resource, as: resource_name, url: session_path(resource_name)) do |f| %>
+    <div class="card-haeder p-2">
+      <h2>Log in</h2>
+    </div>
+    <div class="card-body p-2">
+      <div class="form-group">
+        <%= f.input :email,
+                    required: false,
+                    autofocus: true,
+                    input_html: { autocomplete: "email", class: 'form-control' } %>
+      </div>
+      <div class="form-group">
+        <%= f.input :password,
+                    required: false,
+                    input_html: { autocomplete: "current-password", class: 'form-control' } %>
+      </div>
+      <%= f.input :remember_me, as: :boolean if devise_mapping.rememberable? %>
+    </div>
+
+    <div class="card-footer p-2">
+      <div class="form-group">
+        <%= f.button :submit, "Log in", class: 'btn btn-primary' %>
+      </div>
+    </div>
+  <% end %>
+</div>
```

Comment:
- [ ] Kindly note that in order to meet the project requirement, you will have to create a splash page just as requested by both reviewers above. This splash page is the one that will contain the intro (with the app name) and the two links (Sign up and sign in). At the moment, the root/ default page routes to the login page which is not supposed to be the splash page. Kindly refer to [this comment](https://github.com/chudisoft/budget_manager/pull/1#discussion_r1405260137).